packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -39,7 +40,10 @@ class EventDetails extends CoreElement {
   }
 
   static const defaultTitleText = '[No event selected]';
-  static const defaultTitleBackground = Color(0xFFF6F6F6);
+  static const defaultTitleBackground = ThemedColor(
+    Color(0xFFF6F6F6),
+    Color(0xFF323232),
```

Comment:
Does this color map to a real color in the Material Dark color or some offset of a base?

Reply:
This color is the same as the "lighter" background section in the flame chart (the background of the UI events. I'll change both to use Grey900 +2 from the bottom of Material Dark Theme page. Maybe we should change the background of the plotly chart to match?

Reply:
Seems like the chart and other background sections of the flames charts should match.  Shouldn't all our pages (memory chart too) all be the same?